### ISSUE TITLE
Parameters for Memory Considerations

### ISSUE THREAD

**Issue by woctezuma** [lucidrains/stylegan2-pytorch#23]
In the section called "Memory Considerations", you mention 3 parameters:

```bash
$ stylegan2_pytorch --data /path/to/data \
    --batch-size 3 \
    --gradient-accumulate-every 5 \
    --network-capacity 16
```

I would like to know the equivalent parameters in the original StyleGAN2 repository.

For instance, in [`training_loop.py`](https://github.com/NVlabs/stylegan2/blob/master/training/training_loop.py), I can see:

```python
def training_schedule(
   # [...]
    minibatch_size_base     = 32,       # Global minibatch size.
   # [...]
    minibatch_gpu_base      = 4,        # Number of samples processed at a time by one GPU.
   # [...]
```

```python
def training_loop(
   # [...]
    minibatch_repeats       = 4,        # Number of minibatches to run before adjusting training parameters.
   # [...]
    G_reg_interval          = 4,        # How often the perform regularization for G? Ignored if lazy_regularization=False.
    D_reg_interval          = 16,       # How often the perform regularization for D? Ignored if lazy_regularization=False.
   # [...]
```

I wonder whether:
- `batch-size` is the equivalent of `minibatch_gpu_base`,
- `gradient-accumulate-every` is the equivalent of `minibatch_repeats`,
- `network-capacity` is the equivalent of `minibatch_size_base`.

These are blind guesses based on the default values and the suggested values.

I think it would be interesting to draw a parallel between the two implementations.

**Comment by lucidrains** [comment]
@woctezuma Hi! So there actually isn't any equivalence for `network-capacity` to the original repository, as far as I know.

Otherwise, I believe `batch-size` would be equal to `minibatch_gpu_base` as you had already guessed, and `gradient-accumulate-every` would be equal to `minibatch_size_base / minibatch_gpu_base`

`minibatch_repeats` looks to be something I haven't added to this repository yet, but seems to be in line with this paper https://arxiv.org/abs/1806.07353 . It simply, when training, repeats the same minibatch that many times. It was never mentioned in the Stylegan2 paper, so I don't know how efficacious it is. However, if you find that it works well, I'd be happy to add it as a setting to this repository as well.

**Comment by woctezuma** [comment]
Thank you for your reply!
